src/components/Image/Image.stories.tsx

```diff
@@ -53,6 +59,66 @@ const meta: Meta<typeof Image> = {
 export default meta;
 type Story = StoryObj<typeof Image>;
 
+const imageRow = (opts: any = {}) => {
+  // We'll use this setup function to render all the images in a list item.
+  // Some images display better with a dark background.
+  const styles: any = { textAlign: "center" };
+  const { size = "large", displayValue, sizeBasedOn = "width", id } = opts;
+  if (sizeBasedOn === "width" && size === "default") {
+    styles.width = "100%";
+  } else if (sizeBasedOn === "height" && size === "default") {
+    styles.width = "100%";
+    styles.height = "100%";
+  }
+
+  return (
+    <Box style={styles} key={id}>
+      <Heading id={id} level="h4" size="heading6" text={size} />
+      <Image
+        alt="Alt text"
+        caption={displayValue}
+        size={size}
+        sizeBasedOn={sizeBasedOn}
+        src="//placekitten.com/400/300"
+      />
+    </Box>
+  );
+};
+
+const allVStack = (items) => {
+  return <VStack spacing="l">{items}</VStack>;
+};
+
+const imageSizeValues = [
+  { size: "xxxsmall", display: "32px", id: "ExtraExtraExtraSmall" },
+  { size: "xxsmall", display: "64px", id: "ExtraExtraSmall" },
+  { size: "xsmall", display: "96px", id: "ExtraSmall" },
+  { size: "small", display: "165px", id: "Small" },
+  { size: "medium", display: "225px", id: "Medium" },
+  { size: "large", display: "360px", id: "Large" },
+  { size: "default", display: "100%", id: "Default" },
+];
+
+const sizes = [];
+const sizesBasedOnHeight = [];
+
+for (const imageValue of imageSizeValues) {
+  sizes.push(
+    imageRow({
+      size: imageValue.size,
+      displayValue: imageValue.display,
+      id: imageValue.id,
+    })
+  );
+  sizesBasedOnHeight.push(
+    imageRow({
+      size: imageValue.size,
+      displayValue: imageValue.display,
+      id: imageValue.id,
```

Comment:
This will give you duplicate id issues on the page so add something else to make it unique.